spec/features/admin/csv/download_service_categories_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+feature 'Downloading Service Categories CSV' do
+  before { login_super_admin }
+
+  context 'services has non-empty array attributes' do
+    before do
+      @food = create(:category)
+      @health = create(:health)
+      @jobs = create(:jobs)
+      @service = create(
+        :service,
+        category_ids: [@food.id, @health.id, @jobs.id]
+      )
+
+      @archived_service = create(
+        :service,
+        archived_at: DateTime.now
+      )
+
+      @unarchived_service_at_an_archived_location = create(
+        :service,
+        location: create(:location, archived_at: DateTime.now)
+      )
+
+      visit admin_csv_service_categories_path(format: 'csv')
+    end
+
+    it 'contains the same headers as in the import Wiki' do
+      expect(csv.first).to eq %w[
+      current_parent_categories name id location_name new_category new_subcategory clear_categories
+      ]
+    end
+
+    it 'does not contain archived services' do
+      expect(csv.second).to eq([
+        @service.current_parent_categories, @service.name, @service.id.to_s, @service.location_name, "", "", ""
+      ])
+        
+    end
+
+    it 'does not contain unarchived services that belong to archived locations' do
+      expect(csv.length).to eq(2)
+    end
+
+    it 'contains unarchived services that belong to unarchived locations' do
+      expect(csv.length).to eq(2)
+    end
```

Comment:
These 2 tests with the same expect seem broken. I see how they are not, but it is confusing to look at. The 1 expect that looks at the only data row, on like 35, is really enough to prove this worked, if you also assert that its only 1 rows in that one test. All this is doing is running the before block 2 extra times, really it could just be 1 test with all the asserts about the csv file the comes back.